src/main/preprocessing/code_tracker_handler.py

```diff
@@ -45,16 +48,10 @@ def get_ct_language(data: pd.DataFrame) -> LANGUAGE:
 def handle_ct_file(ct_file: str) -> Tuple[pd.DataFrame, LANGUAGE]:
     log.info(f'Start handling the file {ct_file}')
     ct_df = pd.read_csv(ct_file, encoding=ISO_ENCODING)
-    # It's easier to have all tasks written in snake case because further they will be used as folders names and so on
-    ct_df[CODE_TRACKER_COLUMN.CHOSEN_TASK.value] = ct_df[CODE_TRACKER_COLUMN.CHOSEN_TASK.value].fillna(
-        ''). \
-        apply(lambda t: convert_camel_case_to_snake_case(t))
     language = get_ct_language(ct_df)
     ct_df[CODE_TRACKER_COLUMN.LANGUAGE.value] = language.value
-    ct_df[CODE_TRACKER_COLUMN.AGE.value] = fill_column(ct_df,  CODE_TRACKER_COLUMN.AGE,
-                                                       CODE_TRACKER_COLUMN.AGE.fits_restrictions, DEFAULT_VALUE.AGE)
-    ct_df[CODE_TRACKER_COLUMN.EXPERIENCE.value] = fill_column(ct_df, CODE_TRACKER_COLUMN.EXPERIENCE,
-                                                              CODE_TRACKER_COLUMN.EXPERIENCE.fits_restrictions,
-                                                              DEFAULT_VALUE.EXPERIENCE)
```

Comment:
Is it okay to get rid of checking column restrictions?

Reply:
I deleted it... You can see it: https://github.com/JetBrains-Research/codetracker-data/blob/merge_ct_ati/src/main/preprocessing/code_tracker_handler.py#L54
Or what do you mean?